### ISSUE TITLE
[Quest](Mage) - [16] Investigate the Blue Recluse (id 1920)

### ISSUE THREAD

**Issue by cachocarne** [chromiecraft/chromiecraft#1594]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

This quest seems to be quite messed up. It's supposed to be a lvl 15 quest and one should be able to complete without much problem.

I managed to make it work, but it doesn't feel like it's well scripted. The main problem is that 3-4 mobs can spawn at the same time. 
There are different issues:
- Cantation of Manifestation is not working everywhere in the Inn. We need to find the places to make it work. 
- Even if we get it to work, sometimes nothing happens and no mobs are spawned
- Sometimes we use it in one spot and we get way too many mobs coming ot us which most of times means death or having to run out of the tavern to survive (and mobs will despawn)
- Even if only one is spawned, if then we try to root him by using Frost Nova, other 2 or 3 mobs will appear frozen on the ground, like we trapped them although we could not see them.

Eventually:

I managed to kill one a few times. When using the Trap to capture it, nothing happens. The trap is displayed under the mob with all those lights and stuff, but mob doesn't get trapped and doesn't get accounted 1/3 or anything.
After some time, the Rift Spawns vanishes (steps into the void)

### Expected Blizzlike Behaviour

So this is how it should work:

1 - We grab the items at the mage tower
2 - We go to the Blue Recluse Tavern
3 - We use item 1 Cantation of Manifestation and a Rift Spawn spawns
3 - We damage the Rift Spawn until its hp is down to 1
4 - Rift Spawn gets "stunned"
5 - We use item 2 Chest of Containment Coffers
6 - Rift spawn gets trapped, then we click on the trap and we get it marked as 1/3
7 - We repeat this 2 more times and complete quest

### Source

https://wotlk-twinhead.twinstar.cz/?quest=1920
http://wowhead.com/?quest=1920

### Steps to reproduce the problem

1 - We grab the items at the mage tower
2 - We go to the Blue Recluse Tavern
3 - We use item #1 Cantation of Manifestation and some Rift Spawn spawns
4 - Attempt to kill/stun them and use the traps on them

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/d20f315bd5370761f226460964ae506a97f372c0

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-corldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by cachocarne** [comment]
EDIT: Upon reading many comments in Wowhead, it seems the quest is indeed quite hard to complete and not very clear. It would be nice to be able to spawn 1 mob as indicated on the comments (at the basement or under the stairs) but almost every time there are 3 or 4 mobs appearing and that makes it quite tough.
What **it shouldn't happen** is that we make mobs appear/spawn without using the Scroll but accidentally when using Frost Nova to root one Rift Spawn. If we use AoE dmg, no new Rift Spawns should spawn around us, should they?

**Comment by shinzo69** [comment]
I did the quest several times now and it all depends on the position where u are spawning the mobs via Cantation of Manifestation (item1). If I stand in the middle of the basement im most likely spawning 4-5 mobs but if I do it in a corner of the basement im only spawning 2, which is doable.

Somehow I noticed that after putting the Rifts down to 1hp u cant use Chest of Containment Coffers (item2) because the Rifts are not stunned. I dont know if there is an internal timer for that but in every video I saw they were instantly stunned after dropping to 1HP.

Clips how it should be:
https://youtu.be/91kfcn79zXU?t=40
https://youtu.be/3u6bbzflEDU?t=42

Clip of me doing the Quest:
https://streamable.com/9r9e3g

@Azcobu according to Wowhead comments that's normal behaviour with the Frost Nova.
https://classic.wowhead.com/npc=6492/rift-spawn
